pkg/envtest/server.go

```diff
@@ -81,12 +81,23 @@ type Environment struct {
 	// loading.
 	Config *rest.Config
 
-	// CRDs is a list of CRDs to install
+	// CRDs is a list of CRDs to install.
+	// If both this field and CRDs field in CRDInstallOptions are specified, this
+	// value this ignored.
+	//
+	// Deprecated: Use CRDInstallOptions
 	CRDs []*apiextensionsv1beta1.CustomResourceDefinition
 
 	// CRDDirectoryPaths is a list of paths containing CRD yaml or json configs.
+	// If both this field and Paths field in CRDInstallOptions are specified, this
+	// value this ignored.
+	//
+	// Deprecated: Use CRDInstallOptions
```

Comment:
let's not deprecate these -- they're way more convenient when constructing the object directly -- constructing embedded objects is a pain.

Reply:
Sounds reasonable.